woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2990,13 +2990,19 @@ def test_validate(sample_df):
         invalid_row = ks.from_pandas(invalid_row)
 
     df = df.append(invalid_row)
-    df.ww.init(logical_types={"email": "EmailAddress"})
-    match = "Series email contains invalid email addresses."
+    df["email_2"] = df["email"].fillna("bad_email_2")
+    types = dict(email="EmailAddress", email_2="EmailAddress")
+    df.ww.init(logical_types=types)
 
+    match = "Series email contains invalid email addresses."
```

Comment:
can the bad email addresses be at different index values between the two columns? the concat call should not have a problem with this, and it should still match the expected dataframe--there would just be some additional nan values.
![image](https://user-images.githubusercontent.com/64278226/152064084-b531ce1f-02e8-40b1-8c93-3c2030ba0c59.png)

Reply:
updated